.github/workflows/ci.yml

```diff
@@ -14,4 +14,4 @@ jobs:
     name: Build project and run CI checks
     uses: fingerprintjs/dx-team-toolkit/.github/workflows/build-typescript-project.yml@v1
     with:
-      nodeVersion: 16
+      nodeVersion: 18
```

Comment:
would it be possible to use `nodeVersion` 20?

Reply:
We could, but currently proxy function runs on node 18, so it's better to run CI in the same version